crawler/lookingglass_crawler/settings.py

```
@@ -0,0 +1,194 @@
+        '--disable-renderer-backgrounding',
+        '--disable-backgrounding-occluded-windows',
+        '--disable-background-timer-throttling',
+        '--disable-field-trial-config',
+        '--disable-back-forward-cache',
+        '--disable-hang-monitor',
+        '--disable-prompt-on-repost',
+        '--disable-sync',
+        '--metrics-recording-only',
+        '--no-report-upload',
+        '--use-mock-keychain',
+    ]
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)